orderbook/openapi.yml

```diff
@@ -181,9 +181,9 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Trade"
-  /api/v1/solvable_orders:
+  /api/v1/solver_orders:
```

Comment:
changed because I find this name slightly better

Reply:
I actually like the other name better because it describes the orders where here we imply the use case (which I don't think we have to). Anyways, no big deal.

Reply:
I prefer this new name because everyone I saw solvable orders it gave me the impression that we already know they can be matched.